src/common/snippets/include/snippets/pass/group_norm_tokenization.hpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/pass/graph_rewrite.hpp"
+#include "openvino/pass/pattern/matcher.hpp"
+#include "snippets/pass/tokenization.hpp"
```

Comment:
I hope we can leave only `#include "openvino/pass/pattern/matcher.hpp"`. Could you check please?